crates/rethnet_eth/src/remote/eth.rs

```diff
@@ -175,3 +175,43 @@ where
     let opt = Option::deserialize(deserializer)?;
     Ok(opt.unwrap_or_default())
 }
+
+pub mod eip712 {
```

Comment:
Is this code ever used? If so, consider moving it into a separate file in `src/remote/eth/`

Reply:
addressed in https://github.com/NomicFoundation/hardhat/pull/3998/commits/1f5203b6358c13dc87969761d5db201345b4d639